### ISSUE TITLE
About fully-supervised (oracle) model results

### ISSUE THREAD

**Issue by jsdd25** [wasidennis/AdaptSegNet#51]
Why the oracle resutls of ResNet101 in Table 2 and Table 5 in the paper are different? One is 65.1, and the other is 71.7. In my understanding, both of these results are trained on Cityscapes dataset using the full-supervised model, which should be the same. Thank you very much!

**Comment by wasidennis** [comment]
Table 2 is for GTA5, which is measured on 19 categories, while Table 5 (for SYNTHIA) is measured on only 13 categories. Essentially, they are the same model.

**Comment by Turlan** [comment]
The oracle resutls of ResNet101 in Table 2 is 65.1 as reported. What's your learning rate and training iterations for this setting ?
